extern/test_bmi_py/run_bmi_model.py

```diff
@@ -1,10 +1,32 @@
 from pathlib import Path
 
 import numpy as np
-
+np.set_printoptions(precision=2)
 # This is the BMI LSTM that we will be running
```

Comment:
Not to be pedantic, but...

Reply:
???

Reply:
oh that's on the wrong line somehow. Line 5... it's not LSTM 😝

Reply:
ahh....happy now?